brianlong/www.validators.app
# 181922327 add consensus mods scores

#### What's this PR do?
- Adds consensus mods score to validators

#### How should this be manually tested?
- Run migrations
- Pick a validator that appears both on homepage and /stake-pools page. Run in console: 
```
validator.update consensus_mods: true, security_report_url: 'example.com'
```
- Let demons run for few minutes to calculate new scores 
- Confirm that `validator.score.security_report_score` is 0, and `validator.score.consensus_mods_score` is -2, and these values are included in total_score number.
- Confirm that the same is correctly displayed in the browser, on homepage, show page and stake-pools page.
- Pull this validator via API. Confirm that consensus_mods_score is present in API output, and has correct value.
eg. https://stage.validators.app/validators/2x3pmdwex3s71i8c4VvnHCnYTwXtqPU1LgdX5BLDtN9L?network=mainnet
https://stage.validators.app/validators/9qbqAdFqJK4MhmZPQjbRUxRUi2NTpevAiZ8wj7SiCiTQ?network=mainnet

#### What are the relevant tickets?
- [URL to the PivotalTracker ticket](https://www.pivotaltracker.com/n/projects/2177859/stories/181922327)

#### Task completed checklist:
- [] Is there appropriate test coverage?
- [] Did I take into consideration possible security vulnerabilities?
- [] Are the controllers secure?
- [] Was every important matter documented?

#### Things to watch out for
- [N] Does this PR have any migrations?
- [N] Does this PR add new dependencies?
